### ISSUE TITLE
Does vdebug work with gdb for c/c++/fortran code?

### ISSUE THREAD

**Issue by lervag** [joonty/vdebug#229]
As the title asks "Does vdebug work with gdb for c/c++/fortran code?". After reading docs and searching the interwebs, I am thinking "No". But if it does work, then it seems a very nice replacement for clewn (and similar plugins)!

**Comment by BlackIkeEagle** [comment]
I would say no, Vdebug is targetted at the dbgp protocol.

By default gdb does not speak dbgp if I remember correctly. Maybe you could get it to work with https://github.com/tmc/dbgp/tree/master/gdbproxy wich would be super nice. But not by default

**Comment by BlackIkeEagle** [comment]
Vdebug is targetting the dbgp protocol. Gdb does not speak that by default if I remember correctly.

Maybe you could get it to work with https://github.com/tmc/dbgp/tree/master/gdbproxy which would be super nice ofcourse.

**Comment by lervag** [comment]
I see. Could I propose that one mentions this in the docs under the currently supported languages section?

**Comment by BlackIkeEagle** [comment]
Multi-language DBGP debugger client for Vim (PHP, Python, Perl, Ruby, etc.)

Or is this missing in the vim documentation ?

**Comment by BlackIkeEagle** [comment]
If you get it to work with this gdbproxy I would be glad that its added to the documentation that you can use vdebug also with gdb using that proxy.

**Comment by lervag** [comment]
No, except it is unprecise. The `etc.` implies that there are more languages. But as I stated in my original post, I did get the impression that there was no support for gdb. If this had been clearly stated in the documentation, then I would not have raised this question.

I will test if I can get gdbproxy to work, but then we are already approaching a level of complexity that is just what I was hoping to avoid.
